pkg/kubectl/cmd/get.go

```diff
@@ -220,13 +222,23 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 	}
 
 	// use the default printer for each object
+	printer = nil
+	var lastMapping *meta.RESTMapping
+	w := tabwriter.NewWriter(out, kubectl.TabwriterMinWidth, kubectl.TabwriterWidth, kubectl.TabwriterPadding, kubectl.TabwriterPadChar, kubectl.TabwriterFlags)
```

Comment:
tabwriter translates tabbed columns in input into properly aligned text. Without this, 
```
NAME                                 SECRETS   AGE
default                              1         9m
whatever-you-want                    1         6m
whatever-you-want-really             1         4m
whatever-you-want-really-long        1         4m
whatever-you-want-really-long-name   1         4m
```
becomes something like 
```
NAME    SECRETS   AGE
default 1         9m
whatever-you-want   1         6m
whatever-you-want-really   1         4m
whatever-you-want-really-long   1         4m
whatever-you-want-really-long-name   1         4m
```